ElectronicObserver/Data/Battle/BattleManager.cs

```diff
@@ -48,7 +48,7 @@ public class BattleManager : APIWrapper
 	/// 戦闘結果データ
 	/// </summary>
 	public BattleResultData Result { get; private set; }
-
+	public string TrueRank { get; set; }
```

Comment:
Rename this to `PredictedBattleRank`. Add this summary to describe the property and make sure there's a line of space between this property and the others.
```cs
/// <summary>
/// The battle result api doesn't report SS, so we need to evaluate it manually.
/// </summary>
```